presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java

```diff
@@ -49,4 +51,11 @@ public void testBetaCDF()
     {
         assertQuery("SELECT beta_cdf(nationkey, 0.2, 0.2) from nation where nationkey > 0");
     }
+
+    @Test
+    public void testChiSquaredCDF()
+    {
+        assertQuery("SELECT chi_squared_cdf(acctbal, 2.0) FROM supplier WHERE acctbal > 0.0 AND acctbal < 500.0");
```

Comment:
It didn't seem like you created the Supplier table above. Please can you recheck.

Reply:
Fixed.